### PR TITLE
Properties Pane Text and Screenshot Updates

### PR DESCRIPTION
**Pull request by ConnorLand**

### CODE REVIEW

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 1]
```diff
@@ -28,18 +28,49 @@ An example of data view properties is represented in the image below:
...
-* [Visibility](#visibility)
+* Properties:
+    * [General](#general)
+    * [Data source](#data-source)
+    * [Editability](#editability)
+    * [Visibility](#visibility)
+    * [Common](#common)
+* Styling: 
+    * [Design Properties](#design-properties)
+    * [Common](#common-styling) [PERHAPS SHOULD JUST LINK/SNIPPET TO [Properties Common in the Page Editor](/refguide/common-widget-properties/)]
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: Need to use a snippet to (/refguide/common-widget-properties/) I think.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 2]
```diff
@@ -79,39 +106,23 @@ This property determines how input elements are rendered if they are read-only.
...
+#### 3.2.1 Design Properties Section {#design-properties}
 
-Default: *empty*
+{{% snippet file="/static/_includes/refguide/design-section-link.md" %}} 
 
-### 3.6 Visibility Section {#visibility}
+#### 3.2.2 Common Section {#common-styling}
 
-{{% snippet file="/static/_includes/refguide/visibility-section-link.md" %}}
+[PERHAPS SHOULD JUST LINK/SNIPPET TO [Properties Common in the Page Editor](/refguide/common-widget-properties/)]
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: Need to use a snippet to (/refguide/common-widget-properties/) I think.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 3]
```diff
@@ -28,18 +28,49 @@ An example of data view properties is represented in the image below:
...
+    * [Data source](#data-source)
+    * [Editability](#editability)
+    * [Visibility](#visibility)
+    * [Common](#common)
+* Styling: 
+    * [Design Properties](#design-properties)
+    * [Common](#common-styling) [PERHAPS SHOULD JUST LINK/SNIPPET TO [Properties Common in the Page Editor](/refguide/common-widget-properties/)]
 
-### 3.1 Common Section {#common}
+### 3.1 Properties
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: This organization of one large Properties section with 2 large Properties/Styling sections underneath it creates a lot of # levels. I underestimated what a pain that would be.

I think maybe we should do an easier organization. 

Perhaps just explain in the footer section that the Properties pane is divided by a Properties/Styling toggle, and then link to the sections below as such (note the lack of bullets on Properties and Styling, denoting that they are integer numbered sections and not X.1 sections):

3 Properties:
    * 3.1 [General](#general)
    * 3.2 [Data source](#data-source)
    * 3.3 [Editability](#editability)
    * 3.4 [Visibility](#visibility)
    * 3.5 [Common](#common)
4 Styling:
    * 4.1 [Design Properties](#design-properties)
    * 4.2 [Common](#common-styling)

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: I don't think think that using a snippet in a list is possible/good user experience. If you mean in the section description itself, yes, I would use a snippet.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md (hunk 3 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: What is a footer section? An introduction? Shall we call **Properties/Styling** toggle **Properties** and **Styling** tabs? 
But I agree that structure that you have in the comment as an example is cleaner and has less levels.

Comment (ConnorLand) [reply content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: I've edited the document so it has fewer levels.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md (hunk 2 above)
Comment (ConnorLand) [reply content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: I've introduced a snippet link in this commit. We can use this for all common widget styling properties sections. https://github.com/mendix/docs/pull/6091/commits/dae8885dab170c8b8f01cac30a02470dd928e70f

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 4]
```diff
@@ -10,34 +10,66 @@ tags: ["page", "data view", "widget", "studio pro"]
...
 
-{{< figure src="/attachments/refguide/modeling/pages/data-widgets/data-view/data-view-complex-example.png" >}}
+{{< figure src="/attachments/refguide/modeling/pages/data-widgets/data-view/data-view-complex-example.png"    width="350"   >}}
 
 ## 2 Data View Footer
 
 The footer of the data view is the section at the bottom of the data view that often contains buttons to confirm or cancel changes.  
 
-## 3 Properties
+## 3 Properties Pane
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md:23]: New "intro" section called Properties Pane.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 5]
```diff
@@ -10,34 +10,66 @@ tags: ["page", "data view", "widget", "studio pro"]
...
 
 An example of data view properties is represented in the image below:
 
 {{< figure src="/attachments/refguide/modeling/pages/data-widgets/data-view/data-view-properties.png"   width="250"  >}}
 
-Data view properties consist of the following sections:
+The properties pane is divided into two major sections by a toggle at the top of the pane: **Properties** and **Styling**. Data view properties consist of the following sections:
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md:25]: New intro text explaining the toggle to people.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 6]
```diff
@@ -10,34 +10,66 @@ tags: ["page", "data view", "widget", "studio pro"]
...
 
-* [Common](#common)
+Properties:
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md:27]: New table of contents at the top of the doc with anchor links to the sections below.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md [hunk 7]
```diff
@@ -79,49 +107,33 @@ This property determines how input elements are rendered if they are read-only.
...
+### 4.1 Design Properties Section {#design-properties}
 
-Default: *empty*
+{{% snippet file="/static/_includes/refguide/design-section-link.md" %}} 
 
-### 3.6 Visibility Section {#visibility}
+### 4.2 Common Section {#common-styling}
 
-{{% snippet file="/static/_includes/refguide/visibility-section-link.md" %}}
+{{% snippet file="/static/_includes/refguide/common-styling-props-link.md" %}}
```
Comment (ConnorLand) [inline content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: New common styling snippet. You can use this link in your commits for it. It should work when we merge it all:

{{% snippet file="/static/_includes/refguide/common-styling-props-link.md" %}}

File: static/_includes/refguide/common-styling-props-link.md [hunk 8]
```diff
@@ -0,0 +1,2 @@
+
+The **Common** styling properties allow you to change a widget's class, style, or dynamic class. For more information on class, style, and dynamic classes see [Common Properties](/refguide/common-widget-properties/#common-properties).
```
Comment (ConnorLand) [inline static/_includes/refguide/common-styling-props-link.md]: This is the snippet text that will be displayed for Common styling property sections.

File: content/en/docs/refguide/modeling/pages/page/page-properties.md [hunk 9]
```diff
@@ -9,49 +9,40 @@ tags: ["studio pro", "page", "properties"]
...
-A page parameter is an input that needs to be passed from the calling page, microflow, or nanoflow to the current page. Page parameters determine information that can be reused on the page. For example, if you want to build an **Employee_Details_Edit** page, the page will have a parameter **Employee**. Thus when this page is called an Employee object needs to be passed to it to show the corresponding Employee data.
 
-You can also use multiple page parameters. Multiple page parameters allow you to easily use multiple objects on a page that are not associated with each other. Using multiple page parameters, you can pass multiple arguments when opening a page, the same as with microflows and nanoflows.
+[Styling](#styling):
 
-### 2.3 Designer Section {#designer}
+* [Common](#common-styling)
 
-{{% snippet file="/static/_includes/refguide/designer-properties.md" %}}
+## 3 Properties {#properties}
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/pages/page/page-properties.md:30]: Anchor `#properties` is created for this Properties section.

File: content/en/docs/refguide/modeling/pages/page/page-properties.md [hunk 10]
```diff
@@ -89,50 +80,74 @@ In simple e-commerce applications, the URLs can be configured as follows:
...
+#### 3.7.4 Close Action
 
 Configures the behavior of a button that closes the pop-up page. The default behavior of the pop-up close button is to rollback any changes and close the pop-up window. 
 
 If you want to customize the behavior of the pop-up close button, you can point to a button on the page. When the pop-up close button is clicked, it will then act as if the selected button is clicked. If the selected button is not available the pop-up close button will revert back to the default behavior.
 
 Default: *Default (cancel)*
 
-### 2.7 Usage Section {#usage}
+## 4 Styling {#styling}
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/pages/page/page-properties.md:153]: Anchor `#styling` is created for this Styling section.

File: content/en/docs/refguide/modeling/pages/page/page-properties.md [hunk 11]
```diff
@@ -9,49 +9,40 @@ tags: ["studio pro", "page", "properties"]
 
 This document describes page properties. For details on what pages are for and what kind of widgets can be placed on them, see [Pages](/refguide/pages/).
 
-## 2 Properties {#properties}
+## 2 Properties Pane {#properties-pane}
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/pages/page/page-properties.md:12]: The previous anchor `#properties` is changed to `#properties-pane` for this Properties Pane section.

File: content/en/docs/refguide/modeling/pages/page/page-properties.md [hunk 12]
```diff
@@ -9,49 +9,40 @@ tags: ["studio pro", "page", "properties"]
...
 
 An example of page properties is represented in the image below:
 
 {{< figure src="/attachments/refguide/modeling/pages/page/page-properties/page-properties.png" alt="Page Properties"   width="250"  >}}
 
-Page properties consist of the following sections:
+The properties pane is divided into two major sections by a toggle at the top of the pane: **Properties** and **Styling**. Page properties consist of the following sections:
 
+[Properties](#properties):
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/pages/page/page-properties.md]: Cross-reference is added here for [Properties] via using the anchor `(#properties)`.

File: content/en/docs/refguide/modeling/pages/page/page-properties.md [hunk 13]
```diff
@@ -9,49 +9,40 @@ tags: ["studio pro", "page", "properties"]
...
-### 2.2 Data Section {#data}
-
-#### 2.2.1 Parameters {#parameters}
-
-The list of parameters that this page expects. These parameters can be used by [data views](/refguide/data-view/) with a page parameter data source.
-
-A page parameter is an input that needs to be passed from the calling page, microflow, or nanoflow to the current page. Page parameters determine information that can be reused on the page. For example, if you want to build an **Employee_Details_Edit** page, the page will have a parameter **Employee**. Thus when this page is called an Employee object needs to be passed to it to show the corresponding Employee data.
 
-You can also use multiple page parameters. Multiple page parameters allow you to easily use multiple objects on a page that are not associated with each other. Using multiple page parameters, you can pass multiple arguments when opening a page, the same as with microflows and nanoflows.
+[Styling](#styling):
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/pages/page/page-properties.md]: Cross-reference is added here for [Styling ]via using the anchor `(#styling)`.

File: content/en/docs/refguide/modeling/pages/page/page-properties.md [hunk 14]
```diff
@@ -89,50 +80,74 @@ In simple e-commerce applications, the URLs can be configured as follows:
...
 
-#### 2.7.1 Mark as Used
+### 4.1 Common Section {#common-styling}
 
-You can search for unused items in Studio Pro by pressing <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>F</kbd>. Pages that are only used from Java code will be listed as unused, because Studio Pro cannot look inside Java source code.
-
-By setting the property **Mark as used** to *Yes*, you specify that the document is used implicitly and Studio Pro will no longer list it when searching for unused items.
-
-*Default value*: No
+{{% snippet file="/static/_includes/refguide/common-section-link.md" %}}
```
Comment (Yiyun333) [inline content/en/docs/refguide/modeling/pages/page/page-properties.md:157]: I use here the same snippet that is used for the Common section in the above Properties section. Using two different snippets that refer to the same section in the [Properties Common in the Page Editor](https://docs.mendix.com/refguide/common-widget-properties/#common-properties) doc is a bit confusing to me.

File: content/en/docs/refguide/modeling/pages/data-widgets/data-view.md (hunk 7 above)
Comment (ConnorLand) [reply content/en/docs/refguide/modeling/pages/data-widgets/data-view.md]: We're deciding not to use this "common styling" snippet. Instead we'll just use the "common" snippet.

**Comment by ConnorLand** [comment]
I want to take a look at removing the word "sections" from the section headers in widget documentation, but that will happen at a later time.

**Pull request merged**